contracts/interfaces/IAxelarGMPTransfer.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title IAxelarGMPTransfer
+ * @dev Interface for tracking asset value transfers using General Message Passing (GMP) calls in the Axelar Network.
+ * This interface defines an event that should be emitted when a GMP transfer occurs,
+ * allowing for standardized volume tracking across different implementations.
+ */
+interface IAxelarGMPTransfer {
```

Comment:
update a test contract to make use of this on send and execute for coverage, e.g. `GMPExecutableTest`. The corresponding [test](https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/blob/febcd9fb2b4c0a6db23f5265f3ae9e115a3f2fc6/test/executable/GMPExecutable.js#L109-L110) for the event can be updated as well

Reply:
Yeah, we should do that. @SGiaccobasso , can you run with this?